autobahn/wamp/cryptosign.py

```diff
@@ -137,12 +137,16 @@ def get_bytes(self, size):
         return value
 
     def get_uint32(self):
-        return int.from_bytes(self.get_bytes(4), 'big')
+        return struct.unpack('>I', self.get_bytes(4))[0]
 
     def get_string(self):
         return self.get_bytes(self.get_uint32())
 
 
+def _makepad(l):
+    return ''.join(chr(x) for x in range(1, l + 1))
```

Comment:
I would choose a different name for this arg (`size` maybe?) as it's not that easy to distinguish `1 and l`